Remove ". Linkage error report"

E.g. this is a typical message now:

```
[ERROR] Linkage Checker rule found 1 error. Linkage error report:
Class com.sun.xml.internal.ws.api.BindingID$SOAPHTTPImpl is not accessible;
  referenced by 1 class file
    com.sun.tools.internal.ws.wscompile.WsgenOptions (com.sun:tools:1.8)
  Cause:
    Unknown
```

It can more simply be

```
[ERROR] Linkage Checker rule found 1 error:
Class com.sun.xml.internal.ws.api.BindingID$SOAPHTTPImpl is not accessible;
  referenced by 1 class file
    com.sun.tools.internal.ws.wscompile.WsgenOptions (com.sun:tools:1.8)
  Cause:
    Unknown
```

--- a/enforcer-rules/src/it/bom-project-error/verify.groovy
+++ b/enforcer-rules/src/it/bom-project-error/verify.groovy
@@ -1,7 +1,7 @@
 def buildLog = new File(basedir, "build.log").text.replaceAll("\\r\\n", "\n")
 
 assert buildLog.contains('''\
-[ERROR] Linkage Checker rule found 4 errors. Linkage error report:
+[ERROR] Linkage Checker rule found 4 errors:
 Class org.apache.avalon.framework.logger.Logger is not found;
   referenced by 1 class file
     org.apache.commons.logging.impl.AvalonLogger (commons-logging:commons-logging:1.1.1)

--- a/enforcer-rules/src/it/bom-project-using-spring-repository/verify.groovy
+++ b/enforcer-rules/src/it/bom-project-using-spring-repository/verify.groovy
@@ -12,4 +12,4 @@ assert !buildLog.text.contains(
 
 assert !buildLog.text.contains("NullPointerException")
 
-assert buildLog.text.contains("Linkage error report:")
+assert buildLog.text.contains("Linkage Checker rule found 854 errors:")

--- a/enforcer-rules/src/it/fail-build-for-linkage-errors/verify.groovy
+++ b/enforcer-rules/src/it/fail-build-for-linkage-errors/verify.groovy
@@ -1,7 +1,7 @@
 def buildLog = new File(basedir, "build.log").text.replaceAll("\\r\\n", "\n")
 
 assert buildLog.contains('''\
-[ERROR] Linkage Checker rule found 4 errors. Linkage error report:
+[ERROR] Linkage Checker rule found 4 errors:
 Class org.apache.avalon.framework.logger.Logger is not found;
   referenced by 1 class file
     org.apache.commons.logging.impl.AvalonLogger (commons-logging:commons-logging:1.1.1)

--- a/enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java
+++ b/enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java
@@ -235,7 +235,7 @@ public class LinkageCheckerRule extends AbstractNonCacheableEnforcerRule {
         if (errorCount > 0) {
           String message =
               String.format(
-                  "Linkage Checker rule found %d %s. Linkage error report:\n%s",
+                  "Linkage Checker rule found %d %s:\n%s",
                   errorCount, foundError, LinkageProblem.formatLinkageProblems(linkageProblems));
           String dependencyPaths =
               dependencyPathsOfProblematicJars(classPathResult, linkageProblems);

--- a/enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java
+++ b/enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java
@@ -236,7 +236,7 @@ public class LinkageCheckerRuleTest {
       verify(mockLog, times(2)).error(errorMessageCaptor.capture());
 
       List<String> errorMessages = errorMessageCaptor.getAllValues();
-      Truth.assertThat(errorMessages.get(0)).startsWith("Linkage Checker rule found 112 errors.");
+      Truth.assertThat(errorMessages.get(0)).startsWith("Linkage Checker rule found 112 errors:");
       Truth.assertThat(errorMessages.get(1))
           .startsWith(
               "Problematic artifacts in the dependency tree:\n"
@@ -260,7 +260,7 @@ public class LinkageCheckerRuleTest {
     } catch (EnforcerRuleException ex) {
       // pass
       verify(mockLog)
-          .error(ArgumentMatchers.startsWith("Linkage Checker rule found 1 reachable error."));
+          .error(ArgumentMatchers.startsWith("Linkage Checker rule found 1 reachable error:"));
       assertEquals(
           "Failed while checking class path. See above error report.", ex.getMessage());
     }
@@ -277,7 +277,7 @@ public class LinkageCheckerRuleTest {
     rule.setLevel(EnforcerLevel.WARN);
     rule.execute(mockRuleHelper);
     verify(mockLog)
-        .warn(ArgumentMatchers.startsWith("Linkage Checker rule found 1 reachable error."));
+        .warn(ArgumentMatchers.startsWith("Linkage Checker rule found 1 reachable error:"));
   }
 
   @Test
@@ -440,7 +440,7 @@ public class LinkageCheckerRuleTest {
           "The rule should raise an EnforcerRuleException for artifacts missing dependencies");
     } catch (EnforcerRuleException ex) {
       // pass
-      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 112 errors."));
+      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 112 errors:"));
       assertEquals("Failed while checking class path. See above error report.", ex.getMessage());
     }
   }
@@ -462,7 +462,7 @@ public class LinkageCheckerRuleTest {
     } catch (EnforcerRuleException ex) {
       // pass.
       // The number of errors was 112 in testExecute_shouldFailForBadProjectWithBundlePackaging
-      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 93 errors."));
+      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found 93 errors:"));
       assertEquals("Failed while checking class path. See above error report.", ex.getMessage());
     }
   }

--- a/gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java
+++ b/gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java
@@ -150,7 +150,7 @@ public class LinkageCheckTask extends DefaultTask {
       if (errorCount > 0) {
         getLogger()
             .error(
-                "Linkage Checker rule found {} error{}. Linkage error report:\n{}",
+                "Linkage Checker rule found {} error{}:\n{}",
                 errorCount,
                 errorCount > 1 ? "s" : "",
                 LinkageProblem.formatLinkageProblems(linkageProblems));

